README.md

```diff
@@ -144,7 +144,7 @@ This will install the toml-rb gem on compiling nodes, and grant the appropriate
 ```
 include puppet_operational_dashboards
 ```
-This will install and configure Telegraf, InfluxDB, and Grafana.
+This will install and configure Telegraf, InfluxDB, and Grafana.Please make sure to restart the puppet server after installing the gem.
```

Comment:
This isnt required due to this line :

https://github.com/puppetlabs/influxdb/blob/ee0f7d656fa8813d2ff5ece01ad70cf5abf902fc/manifests/profile/toml.pp#L24C12-L24C47

The service is restarted by virtue of including this class